src/ServiceInsight.Tests/ServiceInsight.Tests.csproj

```diff
@@ -167,8 +169,5 @@
       <Name>ServiceInsight</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup>
-    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
-  </ItemGroup>
```

Comment:
IIRC this speeds up Visual Studio's test discovery. Is there a reason it's been removed?